lib/integrations/awsoidc/eks_enroll_clusters_test.go

```diff
@@ -621,3 +622,29 @@ func (m *mockEnrollEKSClusterClient) PresignGetCallerIdentityURL(ctx context.Con
 }
 
 var _ EnrollEKSCLusterClient = &mockEnrollEKSClusterClient{}
+
+func TestKubeAgentLabels(t *testing.T) {
+	kubeClusterLabels := map[string]string{
+		"priority": "yes",
+		"region":   "us-east-1",
+	}
+	resourceID := uuid.NewString()
```

Comment:
Is it necessary to use UUID generation for testing?

Reply:
We can use whatever value here 👍 
Is it bad to use it? I was trying to use a real value here instead of a static one.